share/completions/reg.fish

```diff
@@ -1,4 +1,24 @@
+function reg # placeholder for testing with wine, will be removed at the end of PR
+    wine reg $argv 2> /dev/null
+end
+
+function __reg_run_reg_safely
+    #if which reg > /dev/null # these lines will be uncommented at the end of PR
+        set -l output (reg $argv | tr -d '\r' | tail --lines +2 | string collect)
+        if not string match -q -r "reg: Invalid syntax*" -- $output
+            set output (echo $output | string replace " " "\n" | string collect)
+            echo $output
+        end
+    #end
+end
+
 function __reg_add_complete_args -a previous_token
+    if test "$previous_token" = add
+        set -l current_token (commandline -toc)
+        __reg_run_reg_safely query $current_token
```

Comment:
For `reg add HKEY_CURRENT_CONFIG` it should suggest:

```
HKEY_CURRENT_CONFIG
HKEY_CURRENT_CONFIG\Software
HKEY_CURRENT_CONFIG\System
```

But now it doesn't do it. How to fix it?

Reply:
It's just `commandline -ct`.

Reply:
> I'll have to check if there's a reason it shouldn't work, but it's unnecessary so the easy workaround is to just leave it off

I suggest return non zero status or warn user in another way that it's doesn't have to work. Or maybe just note about it in man page for `commandline`.